Desktop: Enable 'Alt+Left/Right' for history navigation.

infinite-persistence opened the pull request:
## Why
- Consistent behavior across Web and Desktop.

seanyesmunt commented on ui/effects/use-history-nav.js:
```diff
@@ -0,0 +1,25 @@
+export default function useHistoryNav(history) {
+  useEffect(() => {
+    const handleKeyPress = e => {
+      if (e.altKey && !e.ctrlKey && !e.shiftKey) {
```
This should use `cmd` for mac instead of `alt`

seanyesmunt reviewed (changes requested):
Looks good. Just one comment

infinite-persistence commented:
`Update`
- Rebased
-  Changed
```
      if (e.altKey && !e.ctrlKey && !e.shiftKey) {
```
to
```
      if ((e.metaKey || e.altKey) && !e.ctrlKey && !e.shiftKey) {
```
